apps/store/src/services/storyblok/storyblok.ts

```diff
@@ -306,16 +306,13 @@ export type StoryOptions = {
   version?: StoryblokVersion
 }
 
-export const getStoryBySlug = <StoryData extends ISbStoryData | undefined>(
-  slug: string,
-  { version, locale }: StoryOptions,
-) => {
+export const getStoryBySlug = (slug: string, { version, locale }: StoryOptions) => {
   const params: StoryblokFetchParams = {
     version: version ?? USE_DRAFT_CONTENT ? 'draft' : 'published',
-    resolve_relations: 'reusableBlockReference.reference',
+    resolve_relations: 'reusableBlockReference.reference,categories',
```

Comment:
Shouldn't this be `blog-article.categories`?
```suggestion
    resolve_relations: 'reusableBlockReference.reference,categories',
```